lib/matplotlib/tests/test_cbook.py

```diff
@@ -517,3 +517,57 @@ def test_flatiter():
 
     assert 0 == next(it)
     assert 1 == next(it)
+
+
+class TestFuncParser(object):
```

Comment:
if you insist on writing your own lambdas instead of using numpy, then please test that the lambda functions are correct (no stray typos or the like)